ui/src/containers/deployments/DeploymentLogs.js

```diff
@@ -117,7 +130,14 @@ class DeploymentLogs extends Component {
           />
           <Header
             apiDocs="https://docs.datacater.io/docs/api/deployments/"
-            apiPath={`/deployments/${deployment.uuid}/logs`}
+            apiPath={`/deployments/${deployment.uuid}/logs?replica=${this.state.currentReplica}`}
```

Comment:
I often feel like, these should be routes in react :D So, that view selection happens automatically instead of us having to re-create state management.

Reply:
I agree in general. It's not always possible to turn a frontend URI (or React route) into an API path though.

Shall we approach this in a separate PR/issue?

Reply:
Yes, let us approach in a separate PR.